hooks/services/expense/useCreateExpenses.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+import useMutation from '../useMutation';
+import { useSQLiteContext } from 'expo-sqlite';
+
+//TODO
+const useCreateExpense = (expense: Expense) => {
+  const db = useSQLiteContext();
+  const { data, isLoading, error } = useMutation(mutate);
+
+  async function mutate() {
+    // const { whereString, values } = filtersToString(filters);
+    const result = await db.runAsync(
+      'INSERT INTO Expense (amount, description) VALUES (?, ?)',
+      expense.description,
+      expense.amount,
+    );
+    return result;
+  }
+
+  return { data, isLoading, error };
+};
```

Comment:
I believe we should put the payload on the mutate function 

This is more optimized and will not re-render useMutation functions

```suggestion
const useCreateExpense = () => {
  const db = useSQLiteContext();
  const { data, isLoading, error } = useMutation(mutate);

  async function mutate(expense: Expense) {
    // const { whereString, values } = filtersToString(filters);
    const result = await db.runAsync(
      'INSERT INTO Expense (amount, description) VALUES (?, ?)',
      expense.description,
      expense.amount,
    );
    return result;
  }

  return { data, isLoading, error };
};
```